doc/user-guide/Getting-started.md

```diff
@@ -74,15 +89,29 @@ Registered users wishing to add an existing account to Adium should enter the Mo
 
 #### Domains
 
-To use your system’s domain name instead of localhost, edit the following MongooseIM configuration file: `$REPO/_build/prod/rel/mongooseim/etc/ejabberd.cfg` (where `$REPO` is the repository root).
-Find the line that contains the following: `{hosts ["localhost"] }.` Replace localhost with your domain name, for example: `{hosts, ["example.org"] }`.
+To use your system’s domain name instead of localhost, edit the MongooseIM configuration file: `MongooseIM/_build/prod/rel/mongooseim/etc/ejabberd.cfg`.
+Find and replace the line:
+
+```erlang
+{hosts ["localhost"] }.
+```
+
+using your own hostname, for example:
+
+```erlang
+{hosts ["example.org"] }.
+```
+
 Save the configuration file and restart the MongooseIM server.
-A user’s Jabber ID will then use the domain instead of localhost, for example: `user1@example.org`
+A user’s Jabber ID will then contain the new domain instead of localhost, for example: `user1@example.org`
 
```

Comment:
I would add a warning that changing the list of supported domains won't automatically migrate existing accounts.